src/invoices/InvoiceForm/InvoiceForm.js

```diff
@@ -115,6 +121,8 @@ class InvoiceForm extends Component {
     const isEditPostApproval = IS_EDIT_POST_APPROVAL(initialValues.id, initialValues.status);
     const metadata = initialValues.metadata;
     const approvedBy = get(initialValues, 'approvedBy');
+    const selectedVendor = find(orgs, { id: get(formValues, 'vendorId') });
+    const accountCodes = selectedVendor ? getAccountCodes(selectedVendor) : [];
```

Comment:
wouldn't it be more proper and handy to manage this case inside of `getAccountCodes`? So you don't have to copy-paste that code in every place you want to get `accountCodes`

Reply:
Right suggestion. Thank you